custom-headers/branch-deploy-site-headers

```diff
@@ -0,0 +1,5 @@
+# Prevent search engines from indexing preview sites for branch deploys. See
+# - https://docs.netlify.com/routing/headers/#custom-headers-for-different-branch-or-deploy-contexts
+# - https://developers.google.com/search/docs/crawling-indexing/block-indexing
+/*
+  X-Robots-Tag: noindex
```

Comment:
I don't think the case matters, but the Google documentation example uses title case so I followed that example:
 * https://developers.google.com/search/docs/crawling-indexing/block-indexing

Reply:
It should be fine!